dawn/renderer/components/Editor.js

```diff
@@ -439,6 +439,12 @@ class Editor extends React.Component {
               onClick={this.upload}
               glyph="upload"
             />
+            <TooltipButton
```

Comment:
Makes sense. Originally I was going to make it try to get the file without any protocol (i.e. just assume the robot is there, since SFTP doesn't care if runtime's up), but that can be done later.